webapp/src/components/SampleTable.vue

```diff
@@ -49,13 +50,24 @@ export default {
       }));
     },
   },
-  mounted() {
+  async mounted() {
     this.getSamples();
+    await this.getSampleSchema();
   },
   methods: {
     getSamples() {
       getSampleList();
     },
+    async getSampleSchema() {
+      try {
+        this.sampleSchema = await getSchema("sample");
```

Comment:
```suggestion
        this.sampleSchema = await getSchema("samples");
```

Following my comment above this would then be pluralised